resources/views/auth/user.blade.php

```diff
@@ -83,32 +83,68 @@ function phone_editor(){
                             </td>
                         </tr>
                         @endif
-                        {{-- Neptun code, faculties and workshops --}}
+                        {{-- Other personal information --}}
+                        @if($user->hasPersonalInformation())
+                            <tr>
+                                <th scope="row">@lang('info.place_and_date_of_birth')</th>
+                                <td>
+                                    {{ $user->personalInformation->place_of_birth }},  {{ $user->personalInformation->date_of_birth }}
```

Comment:
Shouldn't these be modifiable?

Reply:
Why? These values won't change, unless the user provided wrong values. In this case, the user can write to system administrators to change them.

Reply:
Right, we could make this function available later, with some review. Will be fine for now.